Source/Particles/WarpXParticleContainer.cpp

```diff
@@ -241,8 +241,12 @@ WarpXParticleContainer::AddNParticles (int /*lev*/,
             pinned_tile.push_back_real(i, 0.0);
         }
 
-        particle_tile.resize(pinned_tile.numParticles());
-        amrex::copyParticles(particle_tile, pinned_tile);
```

Comment:
Interesting. Here, `particle_tile` is defined (a few lines above) as `auto& particle_tile = DefineAndReturnParticleTile(0, 0, 0);` 
@roelof-groenewald Could you confirm that you observed cases in which `particle_tile.numParticles()` is non-zero?

Reply:
Yeap, I can confirm that running this in a simulation with particles already present does give `particle_tile.numParticles() > 0` (although I can't explain why based on your comment above). I had made these changes before when I was using AddNParticles in the MCC routine but then changed the strategy there to use the SmartCreate functionality as we discussed. However, the MCC ionization functionality did work using AddNParticles with the changes in this PR.

Reply:
`DefineAndReturn` defines a new tile if one doesn't exist already, but if there is it just returns the one that's there.